### PR TITLE
[TRUS-1031] - Registration Progress - Summary text

### PR DESCRIPTION
**Pull request by jassalrichy**
Spec amends.
Logic for status tags.
Content amends.

### CODE REVIEW

File: app/views/TaskListView.scala.html [hunk 1]
```diff
@@ -46,4 +46,11 @@ <h2>@messages("taskList.savedUntil.heading")</h2>
 
     @components.task_list(sections)
 
+    @if(isTaskListComplete) {
+        <h2 id="summaryHeading">@messages("taskList.summary.heading")</h2>
+        <p>@messages("taskList.summary.paragraph","Trust Name")</p>
```
Comment (manishsw) [inline app/views/TaskListView.scala.html]: @jassalrichy  recheck as per discussion.

File: test/views/TaskListViewSpec.scala [hunk 2]
```diff
@@ -35,6 +36,24 @@ class TaskListViewSpec extends ViewBehaviours with TaskListViewBehaviours {
   private def sections(answers: UserAnswers) =
     new RegistrationProgress(new TaskListNavigator()).sections(answers)
 
+  private def trustDetailsSectionStatus(answers: UserAnswers) =
+    new RegistrationProgress(new TaskListNavigator()).trustDetailsSectionStatus(answers)
```
Comment (manishsw) [inline test/views/TaskListViewSpec.scala]: @jassalrichy  create the object one time outside of method as it is the same in each method.

File: app/views/TaskListView.scala.html (hunk 1 above)
Comment (jassalrichy) [reply app/views/TaskListView.scala.html]: Amended

File: test/views/TaskListViewSpec.scala (hunk 2 above)
Comment (jassalrichy) [reply test/views/TaskListViewSpec.scala]: Amended.

**Pull request merged**